integration_tests/e2e/abilityToWork.cy.ts

```diff
@@ -16,14 +16,13 @@ context('Ability to work page', () => {
     cy.task('reset')
     cy.task('stubSignIn')
     cy.task('stubAuthUser')
+    cy.task('stubPlpPrisonListPageUi')
```

Comment:
These stubs are new stubs and have been added to pretty much every test.
They stub the PLP UI Prisoner List page; and the front end components API calls